backend/apps/security/tasks.py

```
@@ -0,0 +1,125 @@
+        settings, "GITHUB_REPO_NAME", "diksha78dev/Open-Source-Contribution-Atelier"
+    )
+
+    headers = {
+        "Authorization": f"token {token}",
+        "Accept": "application/vnd.github.v3+json",
+    }
+
+    # 1. Fetch Dependabot alerts
+    alerts_url = f"https://api.github.com/repos/{repo}/dependabot/alerts"
+    try:
+        response = requests.get(alerts_url, headers=headers)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)